doc/api/async_hooks.md

```diff
@@ -195,7 +195,7 @@ The `type` is a string that represents the type of resource that caused
 `init` to be called. Generally it will correspond the name of the resource's
 constructor.
 
-```
+```csv
```

Comment:
The values in this part are separated only by a comma, but yeah I agree on using `text` since it's  not a data for csv/yaml. Will do! Thanks :)